rdkcentral/Lightning-UI-Components
# fix(FocusManager): add zIndex override ability

## Description

Adds the ability to add a `selectedZindex` and `defaultZIndex` for the selected item and non-selected items, respectively.

## References

LUI-1341

## Testing

- Run `yarn build` and copy the `dist` folder into the `@lightningjs/ui-components` directory in the IS `node_modules`.
- Set `this._selectedZIndex = 2;` in `_construct()` for Galery3x4CoverArt, then confirm that the selected Tile in the Gallery3x4CoverArt story is always on top of any non-selected item as you navigate around the gallery. (This will require an IS PR to make this change once this is confirmed to work properly.)

## Automation

N/A

## Checklist
